docs/bugs.json

```diff
@@ -1,4 +1,17 @@
 [
+    {
+        "uid": "SOL-2023-1",
```

Comment:
The UID may have to be adjusted depending on which other important bugfixes get merged first.

Reply:
#14366 is now using `SOL-2023-1` and I switched to `SOL-2023-2`. Let's just make sure to use subsequent numbers in other PRs (and to make sure that #14366 actually gets merged :).